src/azure/cli/commands/arm.py

```diff
@@ -226,6 +252,11 @@ def __call__(self, parser, namespace, values, option_string=None):
     cmd = CliCommand(name, handler, simple_output_query=simple_output_query)
     cmd.arguments.update(set_arguments)
     cmd.arguments.update(get_arguments)
+    if function_arguments:
+        cmd.arguments.update(function_arguments)
+    cmd.arguments.pop('instance', None) # inherited from custom_function(instance, ...)
+    cmd.arguments.pop('parent', None)
+    cmd.arguments.pop('expand', None) # possibly inherited from the getter
```

Comment:
Yes, but then you have multiple ignore entries in the params file for something you can take care of here. Is there any case where you would want the getter part of generic update to retain the expand parameter?

---
In reply to: [75964122](https://github.com/Azure/azure-cli/pull/735#discussion_r75964122) [](ancestors = 75964122)